src/drivers/intel/cavs/hda-dma.c

```diff
@@ -962,6 +962,22 @@ static int hda_dma_data_size(struct dma *dma, int channel, uint32_t *avail,
 	return 0;
 }
 
+static int hda_dma_get_attribute(struct dma *dma, uint32_t type,
+				 uint32_t *value)
+{
+	int ret = 0;
+
+	switch (type) {
+	case DMA_ATTR_ALIGNMENT:
+		*value = PLATFORM_HDA_BUFFER_ALIGNMENT;
+		break;
+	default:
+		ret = -EINVAL;
```

Comment:
It's good practice to add break also after default label.

Reply:
ok